conf/services/conflateAdvOps.json

```diff
@@ -622,6 +630,40 @@
                 },
                 "hoot_key":"merger.creators",
                 "required":"true"
+            },
+            {
+            	"id":"building_review_if_secondary_newer",
+                "elem_type":"bool",
+                "defaultvalue":"false",
+                "description":"If true, any buildings in the secondary layer will be automatically reviewed against matching buildings in the reference layer if they are marked with a more recent date than that of the reference building.",
+                "name":"Always Review If Secondary Feature Newer",
+                "hoot_key":"building.review.if.secondary.newer"
+            },
+            {
+        	"id":"building_date_format",
+        	"elem_type":"string",
+        	"description":"Date format string used by the building date tag value.  See QDateTime::fromString for more details.",
+        	"name":"Date Format",
+        	"defaultvalue":"yyyy-MM-ddTHH:mm",
+        	"hoot_key":"building.date.format",
```

Comment:
I don't love exposing things like date format strings in the UI, but couldn't think of a better way to do it and have it be extensible to all date formats.  There is a bug in the UI where you can't put spaces in option values (opened an issue for it), otherwise I would have dropped the "T" from the default format.  

Another thing to note that I made Curran aware of was that I'm assuming the same date format for both input datasets.  If we find a case where they have different date formats, its an easy change to make.